apps/osim/workflows/not-affected.yml

```diff
@@ -1,25 +1,16 @@
 ---
 name: NOT-AFFECTED
 description: >
-  vulnerability does not affect any Red Hat or community products
-  the flaw was analysed but all affects were resolved as notaffected
-  there is no followup action required after resolving the affects
-priority: 1
+  Vulnerability does not affect any Red Hat or community products
+  the flaw was analysed but all affects were resolved as notaffected.
+priority: 0
 conditions:
-  - has affects
   - affects resolved
   - affects notaffected
 states:
-  - name: DRAFT
+  - name: REJECTED
```

Comment:
I see that my previous comment is not fully applicable after this commit (I am reviewing chronologically commit-wise). Still REJECTED does not mean not affected. Not affected means that we created affects and after analysis we closed all of them with **notaffected** affectedness value. Rejection may mean that we close the flaw draft right away without even creating the affects. The reasons may vary. Usually it would be because we do not ship that thing but also that we already know about this or it is some total spam.